src/modules/Message/utils/tokens/asSafeURL.ts

```diff
@@ -0,0 +1,18 @@
+export function asSafeURL(url: string) {
```

Comment:
Can we have a comment about this util function? Brief input & output example would be helpful :)

Reply:
purpose was not to copy but to create an array from iterator

Reply:
ok